backend/internal/domain/services/task_list_service.go

```diff
@@ -0,0 +1,195 @@
+package services
+
+import (
+	"context"
+	"net/http"
+	"shodo/internal/domain/helpers"
+	"shodo/internal/repository"
+	"shodo/models"
+)
+
+const (
+	kDefaultTaskListTitle = "Shoppings"
```

Comment:
never name constants with the k prefix
constant names are based on the meaning they represent.

In go, we don't put `k` in front of constants the same way we don't prefix interfaces with `i`

Reply:
Renamed